stability-checker/deploy/stability-checker/testing.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+CURRENT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+KYMA_TEST_TIMEOUT=${KYMA_TEST_TIMEOUT:=1h}
+
+readonly CONCURRENCY=5
+# Should be fixed name, it is displayed in TestGrid
+readonly SUITE_NAME="testsuite-all"
+
+# shellcheck disable=SC1090
+source "${CURRENT_PATH}/testing-helpers.sh"
+
+kc="kubectl $(context_arg)"
+
+function main() {
+  echo "----------------------------"
+  echo "- Testing Kyma..."
+  echo "----------------------------"
+
+  cts::check_crd_exist
+
+  cts::delete
+
+  inject_addons_if_necessary
+
+  log::info "- Running Kyma tests"
+  # match all tests
+  # shellcheck disable=SC2086
+  kyma test run \
+                --name "${SUITE_NAME}" \
+                --concurrency "${CONCURRENCY}" \
+                --max-retries 1 \
+                --timeout "${KYMA_TEST_TIMEOUT}" \
+                --watch \
+                --non-interactive
+
+  echo "- Tests results"
+  kubectl get cts  ${SUITE_NAME} -o=go-template --template='{{range .status.results}}{{printf "Test status: %s - %s" .name .status }}{{ if gt (len .executions) 1 }}{{ print " (Retried)" }}{{end}}{{print "\n"}}{{end}}'
```

Comment:
There are commands available in the CLI to achieve similar things, please have a look at "test status" command with the different -o options

Reply:
@a-thaler Unfortunately I couldn't achieve what I need using kymaCLI. The stability-checker gathers statistics of the executed tests. It's done using regexp which matches ```"Test status: %s - %s" .name .status``` .  Using e.g. `kyma test status -owide` provides to us much information but I don't see how can we hook it.